### DISCUSSION TITLE
35292BBA-E703-4CB6-BBBB-247286EFD731

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#93] [General]
# What's New in Celestia 1.5.24

https://celestia.mobi/article?guide=35292BBA-E703-4CB6-BBBB-247286EFD731&lang=en

**Comment by exocritic5457** [comment]
I feel like after this update rolled around, whenever I tap off into space to deselect what I'm focusing on, I accidentally select some TYC star that I can't even see

I can't even tap relatively bright stars without selecting a bunch of others really close to it

**Reply by DaveBowman2001** [reply]
If you want to visit a certain object but can't click on it (due to many unseen stars in the way), you can type that object's name in the Search Bar then simply click Go-to Object afterwards

**Reply by levinli303** [reply]
In the next version, an option will be added to change object selection sensitivity. But if you think it is a bug that incorrect objects are selected, please consider reporting a bug with reproduction steps and expected behaviors.

**Comment by 13940951572** [comment]
希望能继续更新！更新刚多行星…

**Comment by exl001** [comment]
Hello, ever since the first uodate of 2023 celestia's frame rate performance really dropped and nebulaes run really badly especially with molecular clouds....im using an older version: 1.5.9 which is perfect with DSO performance. Apart from that the new tinted illumination, shadows, and globulars look awesome...🙏👍🏼
